columnflow/tasks/plotting.py

```diff
@@ -251,7 +292,7 @@ def store_parts(self):
         return parts
 
     def get_plot_shifts(self):
-        return [self.global_shift_inst]
+        return [self.shift_inst]
```

Comment:
Where does this difference come from?

Reply:
We had to replace the `ShiftTask` with the `PlotShiftMixin` to separate plotting from the `ConfigTask`. Since we do not introduce any shifts during plotting itself, this means that we do not need to separate between global and local shift inst, so we can directly use the shift inst instead of distinguishing the two.
We could generalize this in the future (e.g. if we allow `hist_hooks` to introduce shifts), but I feel we should tackle this when it comes up